sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -29,7 +29,7 @@ public Sampler createSampler(ConfigProperties config) {
     String serviceName = config.getString(SERVICE_NAME_PROPERTY);
     if (serviceName == null) {
       Map<String, String> resourceAttributes = config.getMap(ATTRIBUTE_PROPERTY);
-      serviceName = resourceAttributes.get(SERVICE_NAME_PROPERTY);
+      serviceName = resourceAttributes.get(RESOURCE_ATTRIBUTE_SERVICE_NAME_PROPERTY);
```

Comment:
Sigh.. this fixes an apparent bug, but trying to get the service name from `otel.service.name` or `otel.resource.attributes` is fragile, since the resolved `service.name` sent with exported data may be different from either value.